charts/datadog/templates/cluster-agent-rbac.yaml

```diff
@@ -365,6 +365,41 @@ rules:
   - list
   - watch
 {{- include "orchestratorExplorer-config-crs" . }}
+- apiGroups:
```

Comment:
Can you add a comment on this block that says that these ones are needed for the `datadog.appsec.injector` feature? Maybe also clarify that we need them even when it's disabled to perform the cleanup